/**:
  ros__parameters:
    vendor_id: 2a0b
    product_id: 00b8
